src/github.com/getlantern/grtrack/grtrack.go

```diff
@@ -0,0 +1,58 @@
+// Package grtrack provides a utility that helps check for goroutine leaks.
+package grtrack
+
+import (
+	"bytes"
+	"regexp"
+	"runtime/pprof"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	goroutineNumber = regexp.MustCompile(`goroutine ([0-9]+)`)
+)
+
+type Check func(t *testing.T)
+
+func Start() Check {
```

Comment:
`grtrack` is neat! To be more apparent, we can have `Start` to return an interface with `Check(t)` and `CheckAfter(t, waitTime)`. Current implementation is good enough though.